### PR TITLE
Sonar localhost http

### PR DESCRIPTION
**Pull request by michaelarichard**

### CODE REVIEW

**Comment by mrparkers** [comment]
Could we add a variable to enable / disable sonarqube using http?  Only the local environment is affected by this issue, so I don't want to change how this works for other environments

File: tf/modules/rode/loadpolicy.sh.tpl [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 apk add curl
 
+sleep 120
```
Comment (alexashley) [inline tf/modules/rode/loadpolicy.sh.tpl:4]:
```suggestion
```
`helm_release.rode` should wait for Rode to be ready, is there a need for the `sleep`? Or just leftover from debugging?

File: tf/k8s/main.tf [hunk 2]
```diff
@@ -46,7 +46,7 @@ provider "helm" {
 }
 
 provider "sonarqube" {
-  host              = var.enable_sonarqube ? "https://${module.sonarqube[0].sonarqube_host}" : ""
+  host              = var.enable_sonarqube ? "http://${module.sonarqube[0].sonarqube_host}" : ""
```
Comment (alexashley) [inline tf/k8s/main.tf:49]: Ideally we'd use https here if it's available; for instance, the demo environments should have valid certificates. Maybe a flag to control `https` for SonarQube?

**Comment by mrparkers** [comment]
closing in favor of #69

**Pull request closed without merging**